README.md

```diff
@@ -50,6 +50,10 @@ Install this via pip (or your favorite package manager):
 
 `pip install uiprotect`
 
+## History
+
+This project was split off from the pyunifiprotect because that project changed the license to one which would not be accepted in Home Assistant. This project is committed to keeping the MIT license.
```

Comment:
```suggestion
This project was split off from `pyunifiprotect` because that project changed its license to one that would not be accepted in Home Assistant. This project is committed to keeping the MIT license.
```